docs/source/configuration/overview.mdx

```diff
@@ -598,9 +598,13 @@ supergraph:
 
 </MinVersion>
 
-<ExperimentalFeature />
+<Note>
+
+Beginning with [v1.49.0](https://github.com/apollographql/router/releases/tag/v1.49.0), the router supported enhanced operation signature normalization [experimentally](/resources/product-launch-stages/#experimental-features).
+In [v.1.53.0](https://github.com/apollographql/router/releases/tag/v1.53.0), this configuration became [generally available](/resources/product-launch-stages/#general-availability).
```

Comment:
The config rename will happen in v1.54.0, so we should use that as the version it became GA (and not merge this until then).

Reply:
This is merging into dev, so I think we can/should merge before the release, right?

Reply:
Yes! We'll start 1.54.0 (as a release candidate) today and this will not land in our docs until next Tuesday when we finalize 1.54.0. 😄   I'll merge this to make sure it makes the cut.